package.json

```diff
@@ -63,5 +63,8 @@
     "hooks": {
       "pre-commit": "npm-run-all -ln --parallel lint test"
     }
+  },
+  "dependencies": {
+    "react-ga4": "^2.1.0"
```

Comment:
is it needed here? already added at the lower-level package.json

Reply:
ok ok